.circleci/config.yml

```diff
@@ -4,16 +4,7 @@ jobs:
     docker:
       - image: docker:18.09
     steps:
-      - add_ssh_keys:
-          fingerprints:
-            - "93:22:9e:64:86:b2:d7:23:41:ab:6f:e7:1a:02:35:a8"
-      - run: apk add --no-cache --no-progress make git openssh
-      - run:
-          name: Keyscan Github (HACK) #https://discuss.circleci.com/t/known-hosts-in-circle-2-0/18544
-          command: ssh-keyscan -H github.com >> ~/.ssh/known_hosts
-      - checkout
-      - setup_remote_docker
-      - run: make test-image
+      - run: echo skip
```

Comment:
can you delete this job instead?

Reply:
If I did, the PR gets pending indefinitely. Maybe I need perms.